### PR TITLE
Reference column in documents listings

### LINKED ISSUE #3027: Neue Spalte "Aktenzeichen" bei Dokumentauflistungen
Die bestehenden Dokumentauflistungen sollen mit einer zusätzlichen Spalte "Aktenzeichen", ähnlich wie das bei Dossiers der Fall ist ergänzt werden.

### PR DESCRIPTION
**Pull request by tarnap**
screen shot 2017-06-20 at 14 06 40

Closes  #3027

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ Changelog
 2017.3.0 (unreleased)
 ---------------------
 
+- Add reference in document listings. [tarnap]
```
Comment (phgross) [inline docs/HISTORY.txt]: Add reference **column** in document listings. (same in commit and PR title)

File: opengever/tabbedview/tests/test_document_listing.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                                         document_author='Meter Peter',
+                                         receipt_date=date(2016, 2, 13),
+                                         delivery_date=date(2016, 2, 14),
+                                         public_trial=u'unchecked')
+                                 .within(self.dossier)
+                                 .titled(u'Document 2'))
+
+    @browsing
+    def test_lists_documents(self, browser):
+        browser.login().open(self.repository, view='tabbedview_view-documents')
```
Comment (phgross) [inline opengever/tabbedview/tests/test_document_listing.py]: There is no document listing on a repositoryfolder in GEVER, open the view on the dossier.

File: opengever/tabbedview/tests/test_document_listing.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+        self.repository = create(Builder('repository'))
+
+        self.dossier = create(Builder('dossier')
+                              .within(self.repository)
+                              .titled(u'Dossier'))
+
+        self.document_1 = create(Builder('document')
+                                 .having(document_date=date(2016, 2, 10),
+                                         document_author='Peter Meter',
```
Comment (phgross) [inline opengever/tabbedview/tests/test_document_listing.py]: As a good practice, I would use a non ascii string here.

File: opengever/tabbedview/tests/test_document_listing.py (hunk 2 above)
Comment (tarnap) [inline opengever/tabbedview/tests/test_document_listing.py]: There seems to be one in the `tabbedview_view-documents`

screen shot 2017-06-21 at 15 17 31

screen shot 2017-06-21 at 15 20 20

Comment (deiferni) [inline opengever/tabbedview/tests/test_document_listing.py]: @tarnap this is a screenshot from the personal overview 😉 

what @phgross means is to do

```python 
browser.open(self.dossier, …)
```

Comment (tarnap) [inline opengever/tabbedview/tests/test_document_listing.py]: @deiferni (removed the screenshot of the personal overview)
The first screenshot displays a document listing.
What's wrong with that one?

I know he expects me to do `browser.open(self.dossier, ...)`.
Why that listing and not the other?
Are these different listings?

Comment (tarnap) [inline opengever/tabbedview/tests/test_document_listing.py]: @deiferni @phgross 
How come there's a listing in the repository even if there shouldn't be any?
Is this a bug?

`Documents` inherits from `BaseCatalogListingTab`
https://github.com/4teamwork/opengever.core/blob/master/opengever/tabbedview/browser/tabs.py#L121

`BaseCatalogListingTab` is registered for `ITabbedView`
https://github.com/4teamwork/opengever.core/blob/master/opengever/tabbedview/browser/base_tabs.py#L185

**Pull request merged**